README.md

```diff
@@ -1,2 +1,62 @@
-# invenio-statistics-js
-Invenio-Statistics client library for producing D3 graphs based on statistics.
+## Invenio-Statistics-Js
+
+[![Release](https://img.shields.io/github/tag/inveniosoftware/invenio-search-js.svg)](https://github.com/inveniosoftware/invenio-search-js/releases)
+[![LICENSE](https://img.shields.io/github/license/inveniosoftware/invenio-search-js.svg)](https://github.com/inveniosoftware/invenio-statistics-js/blob/master/LICENSE)
+
+Invenio-Statistics is a reusable client library for integrating charts into modern web apps. All charts are built with D3.js.
+
+## Table of contents
```

Comment:
I think you can remove the `Table of contents` and the `Dependencies` sections (they just take space of the page).